cpp/src/aztec/stdlib/primitives/biggroup/biggroup.test.cpp

```diff
@@ -837,15 +838,16 @@ template <typename TestType> class stdlib_biggroup : public testing::Test {
 };
 
 enum UseBigfield { No, Yes };
-typedef testing::Types<TestType<stdlib::bn254<waffle::UltraComposer>, UseBigfield::No>,
-                       TestType<stdlib::bn254<waffle::TurboComposer>, UseBigfield::No>,
-                       TestType<stdlib::bn254<waffle::StandardComposer>, UseBigfield::No>,
-                       TestType<stdlib::bn254<waffle::UltraComposer>, UseBigfield::Yes>,
-                       TestType<stdlib::bn254<waffle::TurboComposer>, UseBigfield::Yes>,
-                       TestType<stdlib::bn254<waffle::StandardComposer>, UseBigfield::Yes>,
-                       TestType<stdlib::secp256r1<waffle::UltraComposer>, UseBigfield::Yes>,
-                       TestType<stdlib::secp256k1<waffle::UltraComposer>, UseBigfield::Yes>>
-    TestTypes;
+using TestTypes = testing::Types<TestType<stdlib::bn254<honk::StandardHonkComposer>, UseBigfield::No>,
```

Comment:
Trying to prevent these tests from taking forever. Seemed like overkill. Maybe we should split these out into a separate CircleCI job.